src/components/AddAnnonce/Description/index.jsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from 'react';
+import Proptypes from 'prop-types';
+import SDescription from './style';
+
+const Description = ({
+  setDistrict,
+  setCity,
+  setDescribe,
+  setRent,
+  inputDistrict,
+  inputCity,
+  inputDescribe,
+  inputRent,
+}) => {
+  const [isLille, setIsLille] = useState(false);
+
+  const handleChangeDistrict = (evt) => {
+    setDistrict(evt.target.value);
+  };
+  const handleChangeCity = (evt) => {
+    setCity(evt.target.value);
+    if (evt.target.value === 'Lille') {
+      setIsLille(true);
+    } else {
+      setIsLille(false);
+    }
+  };
+  const handleChangeDescribe = (evt) => {
+    setDescribe(evt.target.value);
+  };
+  const handleChangeRent = (evt) => {
+    setRent(evt.target.value);
+  };
+
+  return (
+    <SDescription>
+      <div className="partContainer">
+        <div className="addressCity">
+          <div className="city">
+            <h2>Ville :</h2>
+            <select type="text" value={inputCity} onChange={handleChangeCity}>
+              <option selected value="">
+                {' '}
```

Comment:
empty value is normal ? & useless code "{' '}" 😒

Reply:
Impossible to change with the custom repo

Reply:
Yep-possible, let's find how (I won't repeat this everywhere, but I won't let this pass)